### ISSUE TITLE
PhpMyAdmin post_max_size 2048k

### ISSUE THREAD

**Issue by trilogo-lordee** [laradock/laradock#2839]
### Description:
Good day. The problem is that when creating a new environment, there was a problem with loading the import file into PhpMyAdmin. After trying to figure out the cause, there was a localized such problem. In the container, phpmyadmin-misc.ini is created (or already exists, I unfortunately do not know) with variables that will appear as docker-compose.yml variables in the phpmyadmin environment section. When you try to add them, and restart or re-create the container, the values ​​do not change. phpinfo gives post_max_size 2048K. Unfortunately, I can't say exactly where they are from and where their dependencies are, but this particular place affects the definition of the post_max_size phpmyadmin container. This problem causes the fact that I cannot upload files to the database larger than 2048K. I would like to figure out where phpmyadmin-misc.ini comes from and why these variables are there. Thanks. Below I have given instructions on how to repeat the mistake and how to understand where the change is going.

### Expected Behavior:
Finding variables in Dockerfile or .ini or docker-compose.yml for easy modification and obvious impact.

### Context information:

**Output of `git rev-parse HEAD`**
```
e2d2e9db9ee0a71c48841862e4cee2ea545bdd3a
```
**Output of `docker version`**
```
Client: Docker Engine - Community
 Version:           20.10.3
 API version:       1.41
 Go version:        go1.13.15
 Git commit:        48d30b5
 Built:             Fri Jan 29 14:33:21 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.3
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       46229ca
  Built:            Fri Jan 29 14:31:32 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.3
  GitCommit:        269548fa27e0089a8b8278fc4fc781d7f65a939b
 runc:
  Version:          1.0.0-rc92
  GitCommit:        ff819c7e9184c13b7c2607fe6c30ae19403a7aff
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```
**Output of `docker-compose version`**
```
docker-compose version 1.28.2, build 67630359
docker-py version: 4.4.1
CPython version: 3.7.9
OpenSSL version: OpenSSL 1.1.0l  10 Sep 2019
```
**System info: Mac, Windows or Linux. Include which disto/version**
```
Linux Mint 20.1
```

### Steps to reproduce the issue:

1. Build laradock
2. Go to phpmyadmin container conf dir /usr/local/etc/php/conf.d/phpmyadmin-misc.in i(or copying the /usr/local/etc/php  container folder to your local machine)
```
allow_url_fopen=Off
max_execution_time=${MAX_EXECUTION_TIME}
max_input_vars=10000
memory_limit=${MEMORY_LIMIT}
post_max_size=${UPLOAD_LIMIT}
upload_max_filesize=${UPLOAD_LIMIT}
```
3. Change variable values to real values.(Copying the local config folder or file to /usr/local/etc/php phpmyadmin container)
```
allow_url_fopen=Off
max_execution_time=60
max_input_vars=10000
memory_limit=520M
post_max_size=520M
upload_max_filesize=520M
```
4. Go to phpmyadmin post_max_size will be 520M

**Comment by arhikesha** [comment]
I faced the same problem in the new assembly. The change in docker-compose.yml in the phpmyadmin section added the UPLOAD_LIMIT = 200 variable. I rebuilt the phpmyadmin and php-fpm containers, but the changes did not apply !! But when the workspace container was rebuilt, the changes were applied.

**Comment by trilogo-lordee** [comment]
Okay, I think I found a reason. For me, it is more correct, since it will go in accordance with https://hub.docker.com/r/phpmyadmin/phpmyadmin/. I saw that this image is being used, it has these MEMORY_LIMIT & UPLOAD_LIMIT variables. In commit #2389, this problem was completely solved by changing the config in the DockerFile. I think it's worth moving these variables to the docker-compose.yml and .env file and removing these hardcoded actions in the DockerFile.
